extensions/mistral/src/manager/mistral-inference-manager.ts

```
@@ -0,0 +1,156 @@
+  }
+
+  private async factory(params: { [p: string]: unknown }): Promise<void> {
+    const apiKey = params['mistral.factory.apiKey'];
+    if (!apiKey || typeof apiKey !== 'string') throw new Error('invalid apiKey');
+
+    await this.saveToken(apiKey);
+    await this.registerInferenceProviderConnection({ token: apiKey });
+  }
+
+  dispose(): void {
+    this.connections.forEach(disposable => disposable.dispose());
```

Biome (2.4.11)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)